packages/xo-server-audit/src/index.js

```diff
@@ -135,6 +136,20 @@ class AuditXoPlugin {
     this._cleaners = []
     this._xo = xo
 
+    const {
+      enabled = true,
+      schedule = {
```

Comment:
What if there is a single timezones defined?
Why not destructurate cron and timezone as well?

Reply:
I thought about this, generally, when a user set a timezone, he can also define what cron he want to globally configure its schedule. It's not an edge case for me.